packages/manager/modules/billing/src/components/subscription-tile/subscription-tile.component.js

```diff
@@ -0,0 +1,15 @@
+import controller from './subscription-tile.controller';
+import template from './subscription-tile.html';
+
+export default {
```

Comment:
👍 subscription tile is also used in many places. Can this be a common component? that this be used in any project/service?